packages/block-library/src/tag-cloud/index.php

```diff
@@ -51,5 +51,14 @@ function register_block_core_tag_cloud() {
 			'render_callback' => 'render_block_core_tag_cloud',
 		)
 	);
+
+	register_block_style(
+		'core/tag-cloud',
+		array(
+			'name'         => 'outline',
+			'label'        => __( 'Outline', 'gutenberg' ),
+			'style_handle' => 'outline',
```

Comment:
I don't see a style registered with the handle `outline` in this PR and the styles were added to the existing CSS file, so maybe we don't need that part.